docs/admin/access.rst

```diff
@@ -510,3 +499,10 @@ however re-create them if you delete or rename them.
     Never remove the predefined Weblate groups and users, as this can lead to
     unexpected problems. If you have no use for them, you can removing all their
     privileges instead.
+
+Locking down Weblate
+--------------------
+
+If you experience problems with malice on your Weblate installation,
+you can use :setting:`REQUIRE_LOGIN` to require users to sign in and
+:setting:`REGISTRATION_OPEN` to prevent any new registrations.
```

Comment:
I suggest to move this section up again and rename:

```suggestion
Additional access restrictions
--------------------

If you want to use your Weblate installation in a more private manner,
you can configure the Weblate in a such way that only known users will 
have any access to it. In order to do so you should set 
:setting:`REGISTRATION_OPEN` to ``False`` to prevent registrations of any
new users and set :setting:`REQUIRE_LOGIN` to ``/.*`` to require sign in
to access any page.
```

Reply:
I don't think "private" is a good word as it encompasses security. It also isn't the opposite of "public" there.

Reply:
`less public` than? and s/the Weblate/it/ to remove word repetition...